packages/commons/tests/unit/Utility.test.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Test Utility class
+ *
+ * @group unit/commons/utility
+ */
+import { Utility } from '../../src';
+
+describe('Class: Utility', () => {
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    jest.resetModules();
+  });
+
+  describe('Method: getColdStart', () => {
+
+    test('when called multiple times on the parent class, it returns true the first time, then false afterwards', () => {
+
+      // Prepare
+      const utility = new Utility();
+      const getColdStartSpy = jest.spyOn(utility, 'getColdStart');
+
+      // Act
+      utility.getColdStart();
+      utility.getColdStart();
+      utility.getColdStart();
+      utility.getColdStart();
+      utility.getColdStart();
+    
+      // Assess
+      expect(getColdStartSpy).toHaveBeenCalledTimes(5);
+      expect(getColdStartSpy.mock.results).toEqual([
+        expect.objectContaining({ value: true }),
+        expect.objectContaining({ value: false }),
+        expect.objectContaining({ value: false }),
+        expect.objectContaining({ value: false }),
+        expect.objectContaining({ value: false }),
+      ]);
+
+    });
+
+    test('when called multiple times on a child class, it returns true the first time, then false afterwards', () => {
```

Comment:
What would be the use case of testing the functionality from a child class?

Reply:
The other utilities (Tracer, Logger, etc.) will extend this `Utility` class and use it just like in this test case. I agree that it might be a little redundant but adding one test case was easier/faster than build+pack the new version of `commons` & install it in another package to test it.